ahmad-company-db-model.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE "Company" (
+  "id" INTEGER GENERATED BY DEFAULT AS IDENTITY UNIQUE PRIMARY KEY NOT NULL,
+  "name" VARCHAR UNIQUE,
+  "description" VARCHAR,
+  "website" VARCHAR UNIQUE,
+  "size" VARCHAR,
+  "location" VARCHAR,
+  "industry" VARCHAR,
+  "hidden" BOOLEAN dEFAULT false,
+  "created_at" TIMESTAMP dEFAULT (now()) NOT NULL,
+  "updated_at" TIMESTAMP dEFAULT (now()) NOT NULL
+);
+
+CREATE TABLE "Company_Social_Platform" (
```

Comment:
Why did you split the social platforms to another table? wouldn't it be simpler to store them in the same table:
- intagram_url
- facebook_url 
- .....